yb_migrate/cmd/analyzeSchema.go

```diff
@@ -784,6 +784,16 @@ func analyzeSchema() {
 		utils.ErrExit("failed to write report to %q: %s", reportPath, err)
 	}
 	fmt.Printf("-- find schema analysis report at: %s\n", reportPath)
+
+	//add diagnostic flag check here...
+	utils.InitJSON(exportDir)
```

Comment:
Its okay to name it: `InitJson()`

Can we move the call to `InitJson()` function to somewhere to `root.go` or `main.go`, like we do initialization for logging instead of doing this everytime.

Reply:
Moved to root

Reply:
I will have to change this, during a fresh start, the meta info directory does not exist- attempting to init the local json fails

Reply:
Feel free to restructure the code to cater this behaviour.